### PR TITLE
Add project template

### PR DESCRIPTION
**Pull request by thaJeztah**
closes https://github.com/docker/opensource/issues/44

This adds a basic project template that can be used to set up new open source projects.

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @tiborvass @icecrime @crosbymichael @calavera. It's a quick first draft, so may need some tweaking.

We should probably also move some documents from https://github.com/docker/docker/tree/master/project to this repository (issue triage, review etc)

**Comment by jfrazelle** [comment]
LGTM

File: project-template/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## Reporting security issues
+
+The maintainers take security seriously. If you discover a security issue,
+please bring it to their attention right away!
+
+Please **DO NOT** file a public issue, instead send your report privately to
+[security@docker.com](mailto:security@docker.com).
+
+Security reports are greatly appreciated and we will publicly thank you for it.
+We also like to send gifts&mdash;if you're into Docker schwag, make sure to let
```
Comment (calavera) [inline project-template/README.md]: remove that `&mdash;`?

Comment (thaJeztah) [inline project-template/README.md]: yeah, copied it from the docker/docker readme

**Comment by SvenDowideit** [comment]
@thaJeztah @moxiegirl can we merge this?

**Comment by thaJeztah** [comment]
@SvenDowideit should be good to go, we can tweak after

**Comment by thaJeztah** [comment]
oh! didn't merge; I'll merge this, won't hurt to have it here

**Pull request merged**